vuepress/docs/v6.3.2/docs/getting-started/README.md

```diff
@@ -158,19 +158,19 @@ sudo kubectl create namespace entando
 #### Add Custom Resources
 
 ::: tip Why Custom Resources?
-Standard resources in Kubernetes include things like `Pods`, which are groups of one or more containers, `Services`, the way to call or access your pods, and `Ingresses`, for managing external access to your cluster.
+Standard resources in Kubernetes include things like `Pods`, which are groups of one or more containers, `Services`, the way to call or access pods, and `Ingresses`, for managing external access to clusters.
```

Comment:
Ingresses are defined in a cluster and in a project/namespace. It feels wrong to say "access to clusters" like it's suggesting Ingresses live outside of a cluster. Maybe "cluster resources" would work.

Reply:
understood... i'll change to "managing cluster resources."